pkg/process/config/config.go

```diff
@@ -656,3 +686,37 @@ func setupLogger(loggerName config.LoggerName, logFile string, cfg *AgentConfig)
 		config.Datadog.GetBool("log_format_json"),
 	)
 }
+
+func getDDAgentClient() (pb.AgentClient, error) {
+	// This is needed as the server hangs when using "grpc.WithInsecure()"
+	tlsConf := tls.Config{InsecureSkipVerify: true}
+
+	opts := []grpc.DialOption{
+		grpc.WithConnectParams(grpc.ConnectParams{Backoff: backoff.DefaultConfig}),
+		grpc.WithBlock(),
+		grpc.WithTransportCredentials(credentials.NewTLS(&tlsConf)),
+	}
+
+	target, err := getIPCHost()
+	if err != nil {
+		return nil, err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
+	defer cancel()
+	conn, err := grpc.DialContext(ctx, target, opts...)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return pb.NewAgentClient(conn), nil
+}
+
+func getIPCHost() (string, error) {
+	ipcAddress, err := config.GetIPCAddress()
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%v:%v", ipcAddress, config.Datadog.GetInt("cmd_port")), nil
```

Comment:
I think that we may want to use the `JoinHostPort` built-in for this: https://golang.org/pkg/net/#JoinHostPort